test/fixtures/slow-cloud/web.yml.erb

```diff
@@ -0,0 +1,75 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: web
+  labels:
+    name: web
+    app: slow-cloud
+spec:
+  rules:
+  - host: slow-cloud.example.com
+    http:
+      paths:
+      - backend:
+          serviceName: web
+          servicePort: 80
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: web
+  labels:
+    name: web
+    app: slow-cloud
+spec:
+  ports:
+  - port: 80
+    name: http
+    targetPort: http
+  selector:
+    name: web
+    app: slow-cloud
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: web
+  annotations:
+    shipit.shopify.io/restart: "true"
+    kubernetes-deploy.shopify.io/required-rollout: maxUnavailable
+spec:
+  replicas: 3
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+  template:
+    metadata:
+      labels:
+        name: web
+        app: slow-cloud
+    spec:
+      containers:
+      - name: app
+        readinessProbe:
+          exec:
+            command:
+              - sleep
+              - '4'
+        image: nginx:alpine
```

Comment:
we're trying to use busybox for everything now (with `imagePullPolicy: IfNotPresent` and `command: ["tail", "-f", "/dev/null"]`) so that the suite only pulls one tiny image.